java/test/org/openqa/selenium/bidi/input/DefaultMouseTest.java

```diff
@@ -481,7 +481,7 @@ public void testCanMoveOverAndOutOfAnElement() {
     inputModule.perform(
         windowHandle,
         getBuilder(driver)
-            .moveToElement(redbox, redSize.getWidth() / 1 + 1, redSize.getHeight() / 1 + 1)
```

Comment:
here i'm waiting for `bazel test ` results on my local machine
according to the `moveToElement` signature ,` redSize.getWidth() / 1 + 1` should be `int`, `getWidth` returns `int` and dividing by 1 makes no sense
please, tell me if I'm wrong

Reply:
Tests are passing, so it seems to be working.